src/components/UIShell/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file UI shell entry point.
+ * @copyright IBM Security 2019
+ */
+
+export Content from 'carbon-components-react/lib/components/UIShell/Content';
+
+export CarbonHeader from 'carbon-components-react/lib/components/UIShell/Header';
+export HeaderContainer from 'carbon-components-react/lib/components/UIShell//HeaderContainer';
+export HeaderGlobalAction from 'carbon-components-react/lib/components/UIShell//HeaderGlobalAction';
+export HeaderGlobalBar from 'carbon-components-react/lib/components/UIShell//HeaderGlobalBar';
+export HeaderMenu from 'carbon-components-react/lib/components/UIShell//HeaderMenu';
+export HeaderMenuButton from 'carbon-components-react/lib/components/UIShell//HeaderMenuButton';
+export HeaderMenuItem from 'carbon-components-react/lib/components/UIShell//HeaderMenuItem';
+export HeaderName from 'carbon-components-react/lib/components/UIShell//HeaderName';
+export HeaderNavigation from 'carbon-components-react/lib/components/UIShell//HeaderNavigation';
+export HeaderPanel from 'carbon-components-react/lib/components/UIShell//HeaderPanel';
+export HeaderSideNavItems from 'carbon-components-react/lib/components/UIShell//HeaderSideNavItems';
+
+export Switcher from 'carbon-components-react/lib/components/UIShell/Switcher';
+export SwitcherItem from 'carbon-components-react/lib/components/UIShell/SwitcherItem';
+export SwitcherDivider from 'carbon-components-react/lib/components/UIShell/SwitcherDivider';
+
+export SkipToContent from 'carbon-components-react/lib/components/UIShell/SkipToContent';
+
+export SideNav from 'carbon-components-react/lib/components/UIShell/SideNav';
+export SideNavDetails from 'carbon-components-react/lib/components/UIShell/SideNavDetails';
+export SideNavFooter from 'carbon-components-react/lib/components/UIShell/SideNavFooter';
+export SideNavHeader from 'carbon-components-react/lib/components/UIShell/SideNavHeader';
+export SideNavIcon from 'carbon-components-react/lib/components/UIShell/SideNavIcon';
+export SideNavItem from 'carbon-components-react/lib/components/UIShell/SideNavItem';
+export SideNavItems from 'carbon-components-react/lib/components/UIShell/SideNavItems';
+export SideNavLink from 'carbon-components-react/lib/components/UIShell/SideNavLink';
+export SideNavLinkText from 'carbon-components-react/lib/components/UIShell/SideNavLinkText';
+export SideNavMenu from 'carbon-components-react/lib/components/UIShell/SideNavMenu';
+export SideNavMenuItem from 'carbon-components-react/lib/components/UIShell/SideNavMenuItem';
+export SideNavSwitcher from 'carbon-components-react/lib/components/UIShell/SideNavSwitcher';
```

Comment:
Compare to: https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/UIShell/index.js

The only difference is here we are aliasing `Header` as `CarbonHeader` so it doesn't class with our `Header` component.